plugins/azure/recoveryService/recoveryVaultDiagnosticLogsEnabled.js

```diff
@@ -0,0 +1,62 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Recovery Services Vault Diagnostic Logs Enabled',
```

Comment:
can you update the name to Recovery Services Vault Logging Enabled also update the file name to recoveryVaultLoggingEnabled?